remix/app/components/NewsletterList/index.tsx

```diff
@@ -0,0 +1,25 @@
+import { Newsletter } from '@prisma/client';
+
+interface NewsletterListProps {
+  newsletters: Newsletter[];
+}
+
+export default function NewsletterList({ newsletters }: NewsletterListProps) {
+  return newsletters.length > 0 ? (
+    <ul className="flex flex-col gap-4" data-testid="newsletters-list">
+      {newsletters.map((newsletter) => (
+        <li
+          className="w-[80vw] h-20 cursor-pointer static bg-slate-900 rounded-lg"
+          key={newsletter.id}
+        >
+          <div className="w-[80vw] h-20 p-4 absolute bg-white rounded-lg shadow-lg hover:shadow-2xl transition-all duration-150 hover:-m-2 ease-out hover:ease-in">
+            <h3 className="text-lg font-bold">{newsletter.name}</h3>
+            <span className="line-clamp-1">{newsletter.content}</span>
+          </div>
+        </li>
+      ))}
+    </ul>
+  ) : (
+    <small>No newsletters found.</small>
+  );
```

Comment:
Mnor improvement suggestion as when using the ternary operator it can make the code harder to read 💦 

_Option 1: Use the ternary operator + define a method to render the list_

```javascript
const renderNewsletterList({ newsletters }: Newsletter[]) => {
  return <ul className="flex flex-col gap-4" data-testid="newsletters-list">...</ul>
}

export default function NewsletterList({ newsletters }: NewsletterListProps) {
  return newsletters.length ? renderNewsletterList() : <small>No newsletters found.</small>
}
``` 

_Option 2: Use early return_

```javascript

export default function NewsletterList({ newsletters }: NewsletterListProps) {
  if (!newsletters.length) {
   return <small>No newsletters found.</small>
  }

  return <ul className="flex flex-col gap-4" data-testid="newsletters-list">...</ul>
}

Reply:
Yep, I will go with Option 2 and it makes the code look easier to read 🥹

Improved here: [edc9076](https://github.com/nimblehq/js-framework-benchmark/pull/124/commits/edc90760841fac82ae50c85cae4ba05e3b80a099)